languages/julia/exercises/concept/functions-introduction/.meta/design.md

```diff
@@ -1,7 +1,3 @@
-## Goal
-
-The goal of this exercise is to teach the student the fundamentals of the Concept of functions in Julia, so that they are able to understand the basic structure of Exercism exercises.
```

Comment:
> The reason for this change is that the goal doesn't offer any real information that isn't already somewhere else in the document.

Where should this information go now?

Reply:
I'm fine with reverting it here, where there _is_ some additional information. I just don't want it listed if there is no additional useful information. Is that okay?

Reply:
Sounds good!